plugins/textpattern.md

```diff
@@ -71,13 +76,21 @@ tinymce.init({
 });
 ```
 
-Using the configuration in this example, typing `*`, some text, `*`, then space will result in the format `italic` being applied to the text between the `*` symbols. The same will happen for `**` and `bold`. The third pattern will execute `editor.execCommand('createLink', false, 'https://tiny.cloud')`, which will wrap the text between the `~` symbols in an link that points to `https://tiny.cloud`.
+Using the configuration in this example:
+* `{start: '*', end: '*', format: 'italic'}` - Entering text between`*` and then pressing the **spacebar** will result in the `italic` format being applied to the text between the `*` symbols. 
+* `{start: '**', end: '**', format: 'bold'}` - Entering text between`**` and then pressing the **spacebar** will result in the `bold` format being applied. 
+* `{start: '~', end: '~', cmd: 'createLink', value: 'https://tiny.cloud'}` - This executes `editor.execCommand('createLink', false, 'https://tiny.cloud')`, which will wrap the text between the `~` symbols in an link that points to `https://tiny.cloud`.
 
 #### Block patterns
 
-Block patterns have a `start` and either a `format` or a `cmd` and optional `value` property. They do not have an `end` property. This allows for patterns to be used to either apply a block format or execute a command, optionally with the given value.
+Block patterns must have the following:
+* A `start` 
+* A `format` or a `cmd` 
+* An optional `value` property.
```

Comment:
Again the `value` property only works with `cmd`.

Reply:
Fixed